src/org/traccar/model/Position.java

```diff
@@ -57,10 +58,11 @@
     public static final String KEY_THROTTLE = "throttle";
     public static final String KEY_MOTION = "motion";
     public static final String KEY_ARMED = "armed";
-    public static final String KEY_ACCURACY = "accuracy";
     public static final String KEY_GEOFENCE = "geofence";
     public static final String KEY_ACCELERATION = "acceleration";
     public static final String KEY_DEVICE_TEMP = "deviceTemp";
+    public static final String KEY_OPERATOR = "operator";
+    public static final String KEY_COMMAND = "command";
```

Comment:
It's likely that we should use `KEY_TYPE` instead. In most cases "command" means message type.